src/components/breadcrumbs/Breadcrumb.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import * as _ from 'underscore';
+import { ITitleProps, Title } from '../title/Title';
+import { BreadcrumbLink, IBreadcrumbLinkProps } from './BreadcrumbLink';
+
+export interface IBreadcrumbProps extends React.ClassAttributes<Breadcrumb> {
+  links?: IBreadcrumbLinkProps[];
+  defaultLinkPath?: string;
```

Comment:
i wonder if an optional onClick call back could be nice here, suppose you want to send some analytics or any use case I can not think of.  @GermainBergeron thoughts?